.github/workflows/build-branch.yml

```diff
@@ -3,14 +3,13 @@ name: Build Branch SNAPSHOT
 on:
   workflow_dispatch:
   push:
-    branches-ignore:
+    branches:
       - master
   pull_request_target:
     types:
       - opened
       - reopened
       - synchronize
-      - labeled
```

Comment:
I don't think this should be removed. I misunderstood your initial question on slack.

Reply:
just a note that if we have all of the above options enabled, it would trigger 2 runs when a PR is made, which is when we add a `label` and `opened` a PR.

Reply:
I wonder if this will have an impact on adding the safetobuild label that @filipelautert does when reviewing community PRs.

Reply:
When a safeToBuild label is added, it would trigger the build. I tested it on a community PR.

Reply:
Okay sounds like it is safe to remove then.